src/tests/components/forecastSummary.test.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummary from "../../components/ForecastSummary";
+
+describe("ForecastSummary", () => {
+  const validProps = {
+    date: 1525046400000,
+    description: "Stub description",
+    icon: "800",
+    temperature: {
+      min: 12,
+      max: 11,
+    },
+    onSelect: () => {},
```

Comment:
Since we want to mock the function out here we should use the native jest mock to do it:
```
    onSelect: jest.fn(),
```